src/Lykke.Service.EthereumSignApi/Startup.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using JetBrains.Annotations;
+using Lykke.Sdk;
+using Lykke.Service.EthereumCommon;
+using Lykke.Service.EthereumSignApi.Settings;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+
+namespace Lykke.Service.EthereumSignApi
+{
+    [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
+    public class Startup
+    {
+        public void Configure(IApplicationBuilder app)
+        {   
+            app.UseLykkeConfiguration();
+        }
+
+        public IServiceProvider ConfigureServices(
+            IServiceCollection services)
+        {
+            return services.BuildEthereumServiceProvider<AppSettings>
+            (
+                serviceName: "SignApi",
+                
+                #if ENABLE_SENSITIVE_LOGGING
```

Comment:
In other integration any logging is disabled. Without any option.
Do ethereum requeire this option?

Reply:
Just to make it easier to debug in local and dev environment, logging is disabled by default, so it should not be problem in test and prod envs